src/Components/ArtQuiz/Components/ArtQuizLeftPanel.tsx

```diff
@@ -0,0 +1,5 @@
+import { Text } from "@artsy/palette"
+
+export const ArtQuizLeftPanel = () => {
+  return <Text variant="xl">Art Quiz</Text>
+}
```

Comment:
`non-blocking`: agree we should follow up on consolidating this.

Reply:
right I've gotta get rid of this thing :)